packages/jsii-python-runtime/.gitignore

```diff
@@ -10,3 +10,5 @@ dist/
 
 __pycache__
 .mypy_cache
+
+README.md
```

Comment:
Why ignore?

Reply:
Because it's copied over from the top level directory and thus is a generated file in this directory.